mongo/mongo.go

```diff
@@ -3,58 +3,58 @@ package mongo
 import (
 	"context"
 	"errors"
-	"time"
-
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
 	dpMongoHealth "github.com/ONSdigital/dp-mongodb/v2/health"
 	dpMongodb "github.com/ONSdigital/dp-mongodb/v2/mongodb"
 	"github.com/ONSdigital/dp-permissions-api/apierrors"
+	"github.com/ONSdigital/dp-permissions-api/config"
 	"github.com/ONSdigital/dp-permissions-api/models"
 	"github.com/ONSdigital/log.go/log"
 	"go.mongodb.org/mongo-driver/bson"
 )
 
+const (
+	connectTimeoutInSeconds = 5
+	queryTimeoutInSeconds   = 15
+)
+
 //Mongo represents a simplistic MongoDB configuration, with session and health client
 type Mongo struct {
-	URI                     string
-	Database                string
-	Collection              string
-	Connection              *dpMongodb.MongoConnection
-	Username                string
-	Password                string
-	healthClient            *dpMongoHealth.CheckMongoClient
-	IsSSL                   bool
-	ConnectTimeoutInSeconds time.Duration
-	QueryTimeoutInSeconds   time.Duration
+	Database     string
+	Collection   string
+	Connection   *dpMongodb.MongoConnection
+	healthClient *dpMongoHealth.CheckMongoClient
 }
 
-func (m *Mongo) getConnectionConfig(shouldEnableReadConcern, shouldEnableWriteConcern bool) *dpMongodb.MongoConnectionConfig {
+func (m *Mongo) getConnectionConfig(mongoConf config.MongoConfiguration) *dpMongodb.MongoConnectionConfig {
```

Comment:
A comment on semantics - If the package name already specifies 'mongo' it's not really necessary to repeat it in the types.. For example `dpMongodb.ConnectionConfig` would suffice and be easier to read.
The same could be said the other way round too actually, in the `config` package maybe `config.Mongo` or `config.MongoDB` would be clearer.